<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1002</width>
    <height>735</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>961</width>
      <height>661</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tabShow">
     <attribute name="title">
      <string>Show</string>
     </attribute>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>150</y>
        <width>931</width>
        <height>441</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QTableView" name="tableViewShow"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>721</width>
        <height>31</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Connect Database</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit"/>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonConnectDatabase">
         <property name="text">
          <string>Connect Database</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>241</width>
        <height>41</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Choose Table</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBoxChooseTable"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="gridLayoutWidget">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>50</y>
        <width>471</width>
        <height>92</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label_3">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Write your querry here:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLineEdit" name="lineEditquery"/>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="pushButtonFetchMore">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>600</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Fetch More</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonExecuteQuerry">
      <property name="geometry">
       <rect>
        <x>750</x>
        <y>120</y>
        <width>121</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Execute Querry</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabSale">
     <attribute name="title">
      <string>Sale</string>
     </attribute>
     <widget class="QLabel" name="labelTitleMRR">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>10</y>
        <width>81</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt; font-weight:600;&quot;&gt;MRR&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelMRR">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>40</y>
        <width>81</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt; font-weight:600;&quot;&gt;MRR&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelMRR_2">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>40</y>
        <width>81</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt; font-weight:600;&quot;&gt;SALES&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelTitleMRR_2">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>10</y>
        <width>81</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt; font-weight:600;&quot;&gt;SALES&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabML">
     <attribute name="title">
      <string>ML</string>
     </attribute>
     <widget class="QWidget" name="horizontalLayoutWidget_4">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>450</y>
        <width>781</width>
        <height>31</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayoutSelectDataset_3">
       <item>
        <widget class="QLabel" name="labelModelPath_2">
         <property name="text">
          <string>Model Path:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditModelPath_2"/>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonLoadModel_2">
         <property name="text">
          <string>6. Load Model</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="groupBoxCoefficient">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>150</y>
        <width>271</width>
        <height>291</height>
       </rect>
      </property>
      <property name="title">
       <string>Coefficient</string>
      </property>
      <widget class="QTableView" name="tableViewCoefficient_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>251</width>
         <height>141</height>
        </rect>
       </property>
      </widget>
      <widget class="QWidget" name="gridLayoutWidget_6">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>170</y>
         <width>251</width>
         <height>111</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="2" column="0">
         <widget class="QLabel" name="labelRMSE_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>RMSE:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="labelMAE_2">
          <property name="text">
           <string>MAE:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="labelMSE_2">
          <property name="text">
           <string>MSE:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1" colspan="2">
         <widget class="QLineEdit" name="lineEditRMSE_2"/>
        </item>
        <item row="1" column="1" colspan="2">
         <widget class="QLineEdit" name="lineEditMSE_2"/>
        </item>
        <item row="0" column="1" colspan="2">
         <widget class="QLineEdit" name="lineEditMAE_2"/>
        </item>
        <item row="3" column="1" colspan="2">
         <widget class="QPushButton" name="pushButtonSaveModel_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>5. Save Model</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBoxInput">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>490</y>
        <width>501</width>
        <height>121</height>
       </rect>
      </property>
      <property name="title">
       <string>Input:</string>
      </property>
      <widget class="QWidget" name="gridLayoutWidget_7">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>471</width>
         <height>91</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_7">
        <item row="0" column="0">
         <widget class="QLabel" name="labelAvgAreaIncome_2">
          <property name="text">
           <string>Avg. Area Income:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1" colspan="2">
         <widget class="QLineEdit" name="lineEditAvgAreaHouseAge_2"/>
        </item>
        <item row="0" column="1" colspan="2">
         <widget class="QLineEdit" name="lineEditAvgAreaIncome_2"/>
        </item>
        <item row="1" column="3">
         <widget class="QLabel" name="labelAreaPopulation_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Area Population:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QLabel" name="labelAvgAreaNumberofBedroom_2">
          <property name="text">
           <string>Avg. Area Number of Bedroom:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="4">
         <widget class="QLineEdit" name="lineEditAreaPopulation_2"/>
        </item>
        <item row="0" column="4">
         <widget class="QLineEdit" name="lineEditAvgAreaNumberofBedroom_2"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="labelAvgAreaHouseAge_2">
          <property name="text">
           <string>Avg. Area House Age:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0" rowspan="2">
         <widget class="QLabel" name="labelAvgAreaNumberofRoom_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Avg. Area Number of Room:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1" rowspan="2" colspan="2">
         <widget class="QLineEdit" name="lineEditAvgAreaNumberofRoom_2"/>
        </item>
        <item row="2" column="3" rowspan="2" colspan="2">
         <widget class="QPushButton" name="pushButtonPrediction_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>7. Prediction Housing Price</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBoxOutput">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>490</y>
        <width>271</width>
        <height>71</height>
       </rect>
      </property>
      <property name="title">
       <string>Output:</string>
      </property>
      <widget class="QWidget" name="horizontalLayoutWidget_6">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>251</width>
         <height>41</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="labelPredictionPrice_2">
          <property name="text">
           <string>Prediction Price:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditPredictionPrice_2"/>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>69</y>
        <width>781</width>
        <height>31</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayoutSelectDataset_4">
       <item>
        <widget class="QLabel" name="labelSeclectDataset_2">
         <property name="text">
          <string>Select Dataset:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditSelectDataset_2">
         <property name="text">
          <string>chỗ này ghi querry =&gt; generate ra table tại tableview =&gt; pick các biến rồi train xogn save</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonPickDataset_2">
         <property name="text">
          <string>1. Pick Dataset</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonViewDataset_2">
         <property name="text">
          <string>2. View Dataset</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_7">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>110</y>
        <width>157</width>
        <height>31</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLabel" name="labelTrainingRate_2">
         <property name="text">
          <string>Training Rate:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditTrainingRate_2"/>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>%</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="pushButtonExit">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>580</y>
        <width>256</width>
        <height>23</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>8. Exit</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelTitle">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>10</y>
        <width>481</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:20pt; font-weight:600; color:#55aaff;&quot;&gt;House Pricing Prediction App&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QTableView" name="tableViewDataset">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>150</y>
        <width>501</width>
        <height>291</height>
       </rect>
      </property>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_8">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>110</y>
        <width>381</width>
        <height>31</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="QPushButton" name="pushButtonTrainingModel_2">
         <property name="text">
          <string>3. Training Model</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonEvaluateModel_2">
         <property name="text">
          <string>4. Evaluate Model</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditResultNotidication_2"/>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tabSetting">
     <attribute name="title">
      <string>Setting</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>931</width>
        <height>161</height>
       </rect>
      </property>
      <property name="title">
       <string>Thông tin cá nhân</string>
      </property>
      <widget class="QWidget" name="gridLayoutWidget_3">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>891</width>
         <height>121</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QLabel" name="label_16">
          <property name="text">
           <string>First Name</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="label_19">
          <property name="text">
           <string>Phone:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_17">
          <property name="text">
           <string>Last Name</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_18">
          <property name="text">
           <string>Starts Day</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QLabel" name="label_20">
          <property name="text">
           <string>Gmail:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QLabel" name="label_21">
          <property name="text">
           <string>Positions:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="lineEdit_2"/>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="lineEdit_3"/>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="lineEdit_4"/>
        </item>
        <item row="0" column="3">
         <widget class="QLineEdit" name="lineEdit_5"/>
        </item>
        <item row="1" column="3">
         <widget class="QLineEdit" name="lineEdit_6"/>
        </item>
        <item row="2" column="3">
         <widget class="QLineEdit" name="lineEdit_7"/>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>180</y>
        <width>931</width>
        <height>171</height>
       </rect>
      </property>
      <property name="title">
       <string>Privacy</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1002</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSystems">
    <property name="title">
     <string>Systems</string>
    </property>
    <addaction name="actionConnect_to_Database"/>
    <addaction name="actionExport_CSV"/>
   </widget>
   <addaction name="menuSystems"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionConnect_to_Database">
   <property name="text">
    <string>Connect to Database</string>
   </property>
  </action>
  <action name="actionImport_CSV">
   <property name="text">
    <string>Import CSV</string>
   </property>
  </action>
  <action name="actionExport_CSV">
   <property name="text">
    <string>Export CSV</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
